public/app/percona/shared/websockets/notifications.types.ts

```diff
@@ -0,0 +1,35 @@
+import { Severity } from '../core';
+
+export interface CheckStreamData {
+  error: {
+    code: number;
+    message: string;
+    details: [
+      {
+        type: string;
+      }
+    ];
+  };
+  result: {
+    topic: string;
+    results: [
+      {
+        summary: string;
+        description: string;
+        severity: keyof typeof Severity;
+        labels: {
+          additionalProp1: string;
```

Comment:
I think these are just examples, we probably want `labels: {[key: string]: string | number}`

Reply:
hmm I already changed it but didn't push it for some reason...corrected